rcl/src/rcl/graph.c

```diff
@@ -52,7 +52,7 @@ rcl_get_topic_names_and_types(
     &rcutils_allocator,
     no_demangle,
     topic_names_and_types
-  );
+    );
```

Comment:
Please avoid changes to code not touched by this PR.

Reply:
Removed the files that weren't part of my changes.